### ISSUE TITLE
Update on Requirements and Sequence Diagrams

### ISSUE THREAD

**Issue by gktpmuhammed** [bounswe/2021SpringGroup1#117]
Since requirements should be declarative and specific and based on the feedback from our TA Egemen, we need to change 1.1.1.1.3,  1.1.1.1.4 and non-functional requirements. Also it seems that there is a little confusion on user and system requirements. So, we need to check and replace the places of some requirements.

**Comment by gktpmuhammed** [comment]
User privacy, Community privacy and Event Privacy was under nonfunctional requirements but it needs to be in functional requirements. So I moved them to correct places. 
I also deleted 1.2.5.3 and moved 1.2.5.4 and 1.2.5.5 one place up. Lastly I made changes on some requirements to make the sentences informative.

**Comment by gktpmuhammed** [comment]
Also these changes need a review from team members and we need to look [mockups](https://github.com/bounswe/2021SpringGroup1/wiki/Mockup-Scenarios) again to correct Acceptance Criteria sections.

**Comment by MrLebowski42** [comment]
I assigned myself on this issue. I will add some other requirements to flesh out the community functionalities.

**Comment by MrLebowski42** [comment]
I made some significant modification the requirements. New requirement page is [here](https://github.com/bounswe/2021SpringGroup1/wiki/Fixed-Requirements-(-Draft)).
To summarise the changes I've made:
- I moved most of the system requirements to the user side.
- I removed the requirements related to Direct Messaging, as it was not included in the design nor discussed enough. ( Maybe we can add it later.)
- I tried to flesh out most of the existing requirements.
- I added requirements on Post and Post Templates. (under 1.1.6 ,1.1.7 )
- I added the system requirements under 1.2.2 , 1.2.4 , 1.2.5.
- I merged some of the sections together for easy reading later on.

I may stress that this document needs some review so I am not deleting the existing requirement page. It is also not final so any contribution is appreciated.
